src/utils/const.ts

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line operator-linebreak
+export const API_URL =
+  'https://mate-academy.github.io/react_people-table/api/people.json';
+
+export const HOME = '/';
+export const PEOPLE_LIST = '/people';
```

Comment:
```suggestion
export const PATHS = {
  home: '/',
  people: '/people',
} as const;
```
You can use object instead, to avoid a lot of constants

Reply:
Thank you. Done